packages/restish/build.sh

```diff
@@ -3,9 +3,9 @@ TERMUX_PKG_DESCRIPTION="A CLI for interacting with REST-ish HTTP APIs"
 TERMUX_PKG_LICENSE="MIT"
 TERMUX_PKG_LICENSE_FILE="LICENSE.md"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION="0.21.0-cifix"
+TERMUX_PKG_VERSION="0.21.0-cifix.1"
```

Comment:
That isn't exactly a valid version.
*And unfortunately* it's interpreted as later than 0.21.0, so we're in a bind when it comes to actually fixing it.

```suggestion
TERMUX_PKG_VERSION="0.21.0+cifix.1"
```
Using a `+` here *should* work.
###### https://www.debian.org/doc/debian-policy/ch-controlfields.html#version